src/JSON.jl

```diff
@@ -1,3 +1,5 @@
+VERSION >= v"0.4.0-dev+6521" && __precompile__()
```

Comment:
Not critical, but I've been doing `isdefined(Base, :__precompile__) && __precompile__()`, looks a little less messy